addons/zeus/functions/fnc_ui_groupSide.sqf

```diff
@@ -128,7 +128,7 @@ private _fnc_onConfirm = {
     private _unit = effectiveCommander (attachedTo _logic);
     private _side = [west,east,independent,civilian] select (GETVAR(_display,newSide,GETVAR(_display,oldSide,0)));
 
-    [_unit, _side] call FUNC(moduleGroupSide);
+    [_unit, _side, _logic] call FUNC(moduleGroupSide);
     deleteVehicle _logic;
```

Comment:
The UI modules actually get deleted at the end of their respective UI function (if there's no reason to pass the module on to the further function).